### DISCUSSION TITLE
Modding

### DISCUSSION THREAD

**Discussion by Wojtek9388** [SebLague/Digital-Logic-Sim#197] [Ideas]
I want to make a group of a few people and we make a discord server and create mods for this and make videos for other people to learn how to make mods for this

**Comment by zedonme** [comment]
This is a good idea. I would be very interested in such a resource, however I am not good enough at modding to be making the tutorials so I can't help get this started.

**Reply by Maxludus** [reply]
How can I import this to unity editor to mod it?

**Reply by lyranie** [reply]
1. clone the project:
```bash
git clone https://github.com/SebLague/Digital-Logic-Sim.git
```
or using the github cli
```bash
gh repo clone SebLague/Digital-Logic-Sim
```

or alternatively download the zip and extract it

2. download and install [unity](https://unity.com/download)
3. open unity hub
4. click "Add" and select the cloned repository folder
